apps/passport-client/components/screens/HomeScreen/HomeScreen.tsx

```diff
@@ -50,32 +51,30 @@ export function HomeScreenImpl(): JSX.Element | null {
   const self = useSelf();
   const navigate = useNavigate();
   const dispatch = useDispatch();
+  const loadedIssuedPCDs = useLoadedIssuedPCDs();
 
-  const pcdCollection = usePCDCollection();
   const [searchParams, setSearchParams] = useSearchParams();
   const defaultBrowsingFolder = useMemo(() => {
     const folderPathFromQuery = decodeURIComponent(
       searchParams.get(FOLDER_QUERY_PARAM) ?? ""
     );
-    if (!folderPathFromQuery) {
+    if (!folderPathFromQuery || folderPathFromQuery === "null") {
```

Comment:
`"null"` is a bit strange to me - is it possible for the creator of the url to omit the folder path query string parameter altogether if it's not defined or is null or empty?